CHANGELOG.md

```diff
@@ -174,7 +193,10 @@ _No release_
 
 ## [2.1] - 2020-10-05
 
-[unreleased]: https://github.com/TACC-Cloud/hazmapper/compare/v2.16...HEAD
+[unreleased]: https://github.com/TACC-Cloud/hazmapper/compare/v2.19...HEAD
```

Comment:
Why  are we listing unreleased branches here? If it is an unreleased WIP, I don't see why it would be added to the change log. Am I missing something?